tests/test_clean.py

```diff
@@ -305,11 +505,44 @@ def test_attributes_list():
 
     text = u'<a href="/foo" title="blah">example</a>'
     assert (
-        bleach.clean(text, tags=TAGS, attributes=ATTRS) ==
+        clean(text, tags=TAGS, attributes=ATTRS) ==
         u'<a title="blah">example</a>'
     )
 
 
+@pytest.mark.parametrize('data, kwargs, expected', [
+    # javascript: is not allowed by default
+    (
+        '<a href="javascript:alert(\'XSS\')">xss</a>',
+        {},
+        '<a>xss</a>'
+    ),
+
+    # File protocol is not allowed by default
+    (
+        '<a href="file:///tmp/foo">foo</a>',
+        {},
+        '<a>foo</a>'
+    ),
```

Comment:
Previously, this tested `feed:file:///tmp/foo`. I changed that to `file` because that seemed more helpful.

Reply:
I think it'd still be useful to have the double protocol case in addition to this one to make sure we aren't just stripping the first invalid protocol.

Reply:
I like this idea. I'll look into this in a followup PR.